client/ayon_core/plugins/publish/integrate_review.py

```diff
@@ -0,0 +1,102 @@
+import os
+
+import pyblish.api
+import ayon_api
+from ayon_api.server_api import RequestTypes
+
+from ayon_core.lib import get_media_mime_type
+from ayon_core.pipeline.publish import get_publish_repre_path
+
+
+class IntegrateAYONReview(pyblish.api.InstancePlugin):
+    label = "Integrate AYON Review"
+    # Must happen after IntegrateNew
+    order = pyblish.api.IntegratorOrder + 0.15
+
+    def process(self, instance):
+        project_name = instance.context.data["projectName"]
+        src_version_entity = instance.data.get("versionEntity")
+        src_hero_version_entity = instance.data.get("heroVersionEntity")
+        for version_entity in (
+            src_version_entity,
+            src_hero_version_entity,
+        ):
+            if not version_entity:
+                continue
+
+            version_id = version_entity["id"]
+            self._upload_reviewable(project_name, version_id, instance)
+
+    def _upload_reviewable(self, project_name, version_id, instance):
+        ayon_con = ayon_api.get_server_api_connection()
+        major, minor, _, _, _ = ayon_con.get_server_version_tuple()
+        if (major, minor) < (1, 3):
+            self.log.info(
+                "Skipping reviewable upload, supported from server 1.3.x."
+                f" User server version {ayon_con.get_server_version()}"
```

Comment:
It being referred to as the "User" server version seems a bit odd?
```suggestion
                f" Current server version {ayon_con.get_server_version()}"
```